parquet/src/file/metadata.rs

```diff
@@ -353,6 +377,15 @@ impl RowGroupMetaDataBuilder {
         self
     }
 
+    /// Sets the sorting order for columns
+    pub fn set_sorting_columns(
+        mut self,
+        value: Option<&Vec<SortingColumnMetaData>>,
+    ) -> Self {
+        self.sorting_columns = value.cloned();
```

Comment:
```suggestion
        value: Option<Vec<SortingColumnMetaData>>,
    ) -> Self {
        self.sorting_columns = value;
```

Passing in a reference only to copy it is a little unusual